packages/eas-cli/src/utils/queries.ts

```diff
@@ -119,7 +119,9 @@ async function paginatedQueryWithSelectPromptInternalAsync<
 
   const valueOfUserSelectedListItem = await selectAsync<string>(
     promptOptions.title,
-    selectionPromptListItems
+    selectionPromptListItems,
+    undefined,
+    promptOptions.selectPromptWarningMessage
```

Comment:
I prefer not to pass `undefined` explicitly to a function. This is not readable (without a label), you have to find the function definition to figure out why this is not passed.
Instead, if there are a few optional params, you can group them in an (optional) object.